### PR TITLE
fix: fit channel avatars to consistent size

### PR DESCRIPTION
**Pull request by thisiszinger**
| Before | After |
|---|---|
| Capture-2024-10-02-114626 | Capture-2024-10-02-114719 |

### CODE REVIEW

**Comment by pugson** [comment]
this doesn't happen for me. are you sure you're not overriding some css properties?

**Comment by thisiszinger** [comment]
I'm not touching any styles, just rendering the `<FarcasterEmbed />` directly. Have you tried specifically with the [/latenightcrew](https://warpcast.com/~/channel/latenightcrew) channel? It depends on the channel avatar, some are square so they just work.

File: src/styles.css [hunk 1]
```diff
@@ -254,9 +249,18 @@
...
 }
 
-.farcaster-embed-channel-avatar {
-  display: inline-block;
+.farcaster-embed-channel-avatar-container {
+  width: 16px;
+  height: 16px;
   border-radius: 3px;
+  overflow: hidden;
+  background-color: red;
```
Comment (pugson) [inline src/styles.css]:
```suggestion
```

**Review by pugson — CHANGES_REQUESTED** [review]

File: src/components/cast.tsx [hunk 2]
```diff
@@ -181,13 +181,9 @@ export function CastEmbed({
...
               {cast.tags[0].imageUrl && (
-                <img
-                  src={cast.tags[0].imageUrl}
-                  alt={cast.tags[0].name}
-                  width={16}
-                  height={16}
-                  className="farcaster-embed-channel-avatar"
-                />
+                <div className="farcaster-embed-channel-avatar-container">
+                  <img src={cast.tags[0].imageUrl} alt={cast.tags[0].name} className="farcaster-embed-channel-avatar" />
```
Comment (pugson) [inline src/components/cast.tsx]:
```suggestion
                  <img src={cast.tags[0].imageUrl} alt={cast.tags[0].name} className="farcaster-embed-channel-avatar" width={16} height={16} />
```

**Review by pugson — CHANGES_REQUESTED** [review]

**Review by pugson — APPROVED** [review]

**Pull request merged**